deploy_carvel/concierge/schema-openapi.yaml

```diff
@@ -0,0 +1,137 @@
+openapi: 3.0.0
```

Comment:
We can remove this file from git but let the script generate it each time its run.